### ISSUE TITLE
Transporters From Engineering - Transproter Malfunction Debuff

### ISSUE THREAD

**Issue by Malozar** [SunstriderEmu/BugTracker#2742]
**Describe the bug**
When using  [Ultrasafe Transporter: Toshley's Station] (and presumable the one to area 52 aswell), there is a chance to get the debuff [Transporter Malfunction], of which the flavor text is "Looks like your pattern matrix got mixed up with the last person to use the transport pad."

I clearly remember this back from TBC, and youre supposed to become a random race, whether its actually based on previous users or not is doubtful, but im 100% certain a race was given to you at random when recieving this debuff. However, it seems that when given the debuff on Sunstrider, you only can become a female bloodelf.

**To Reproduce**
Use [Ultrasafe Transporter: Toshley's Station] or [Dimensional Ripper - Area 52]. Gain the [Transporter Malfunction] debuff.

Item: https://www.wowhead.com/item=30542/dimensional-ripper-area-52
Item: https://www.wowhead.com/item=30544/ultrasafe-transporter-toshleys-station
Spell: https://www.wowhead.com/spell=36897/transporter-malfunction

**Expected behavior**
The way/randomness of the given buffs seems to be as its supposed to, but it should be expected to (most likely) see a different race each time the Transporter Malfunction debuff is applied.

**Screenshots/videos**
https://wow.zamimg.com/uploads/screenshots/normal/49392-ultra
safe-transporter-toshleys-station-looks-like-your-pattern-matrix-got-mixed-.jpg
https://www.wowhead.com/item=30544/ultrasafe-transporter-toshleys-station#screenshots:id=116433

https://wow.zamimg.com/uploads/screenshots/normal/177871-ultrasafe-transporter-toshleys-station-tauren-rogue-on-mechanostrider-is-srs-bsn.jpg

**Additional context**
Long story short, the only bug is just the fact that the same race is given every time you get the debuff.

**Comment by Distorted39** [comment]
I've noticed this aswell with Dimensional Ripper - Area 52.
I also remember the races change (between the races you can get with this debuff) when you switch zones or eating things that change your race, for example Savory Deviate Delight or Noggenfogger Elixir.

**Comment by Jerrythemagicdog** [comment]
Fixed some time ago, thanks for the issue.
